source/configuring_domain/03_managing_tracking/third_party_analytics.rst

```diff
@@ -1,35 +1,61 @@
-Third-party Analytics
+Third-party analytics
 =====================
 
-The monitoring tools included in Opendatasoft are centered about API calls and usages; this is useful to gather extremely precise information about how
-your data is consumed, but can be limiting if you try to measure the audience of your portal in a more broad manner, such as your referral sources,
-your visitor's geographical origin, or the time spent on your portal.
+The monitoring tools provided by Opendatasoft focus on API calls and usages, which is useful to gather precise information on how
+data is consumed (see :doc:`Analyzing domain usage </managing_domain/03_analyzing_domain_usage/index>`). But it can be limiting when trying to measure more broadly the audience of a portal (i.e. referral sources, visitors geographical origin, time spent on the portal, etc.).
 
-Some third-party services are specialized in measuring audience, and can give this kind of information. Opendatasoft supports integrations with **Google Analytics**
-and **XiTi**.
+Third-party services, specialized in measuring audience, can give additionnal analytics such as those mentioned above. Opendatasoft supports integrations with Google Analytics, and XiTi and SmartTag by AT Internet.
 
-If you need support for another analytics solution that is not mentioned here (such as Piwik or SmartTag), please contact us.
+All third-party analytics integrations are activated and configured from Configuration > Tracking in the back office.
+
+
+.. admonition:: Important
+   :class: important
+
+   For most integrations, Opendatasoft must be involved to allow the activation of a third-party service from the back office.
```

Comment:
This is only for two out of four (XiTi and SmartTag), maybe we can just mention them.
e.g.
> For XiTi and SmartTag integration, Opendatasoft must be involved for the activation process, before the service can be configured in the back-office.

Reply:
Also the doc doesn't mention Piwik (or its new name Matomo) anymore, which we can currently add with a feature flag, but we'll soon add it officially I think anyway.